router/pkg/config/config.go

```diff
@@ -983,6 +983,12 @@ type PluginRegistryConfiguration struct {
 	URL string `yaml:"url" env:"URL" envDefault:"cosmo-registry.wundergraph.com"`
 }
 
+type IntrospectionConfiguration struct {
+	Enabled            bool   `yaml:"enabled" envDefault:"true" env:"INTROSPECTION_ENABLED"`
+	SkipAuthentication bool   `yaml:"skip_authentication" envDefault:"false" env:"INTROSPECTION_SKIP_AUTHENTICATION"`
+	Token              string `yaml:"token" env:"INTROSPECTION_TOKEN"`
+}
```

Comment:
Could this be like the below example? Is the non-token authentication even required by the ticket? We could potentially alleviate some of the concerns with duplicating the operationkit and PO and etc if all requests are authenticated _somehow_
```yml
introspection:
  enabled: true
  authentication_mode: full | token | skip
  token: "wasd" (required in JSON schema if authentication mode == token)
```